### ISSUE TITLE
bullet points not working inside monster stats blocks

### ISSUE THREAD

**Issue by miguel-pinto-photobox** [stolksdorf/homebrewery#64]
**Browser Type/Version**: Chrome

**Operating System**: Win 7

**Issue Description**: the bullets are not working inside the monsters stats block. It doesn't show the bullets

**Markdown code to reproduce**:

```
___
> ***Uga Buga.*** bla bla bla:
> * bullet 1
> * bullet 1
> * bullet 1
```

**Comment by kkragenbrink** [comment]
This seems to be intentional behaviour.  It's actually written into the CSS to have this effect.

If you want to change it, you can add the following to your code:

```html
<style type="text/css">
.phb hr+blockquote li { list-style-type: disc; }
</style>
```

**Comment by stolksdorf** [comment]
It's common in the PHB to have lists with no markings, but begin with bolded terms (as in the top monster stat blocks). I wrote the styling to default to that. It should be a simple rule change to specify that the list must be right after an `hr` to have that effect.

**Comment by stolksdorf** [comment]
_Finally_ fixed this in v2.2.1
